CHANGELOG.md

```diff
@@ -30,6 +30,12 @@
 - Upon panic, the terminal is now reset to canonical mode.
 - Explicit error upon failure of vsock device creation.
 - The failure message returned by an API call is flushed in the log FIFOs.
+- Insert virtio devices in the FDT in order of their addresses sorted from
+  low to high.
+- Enforce the maximum length of the network interface name to be 16 chars as
+  specified in the Linux Kernel.
+- Changed the vsock property `id` to `vsock_id` so that the API client can be
+  successfully generated from the swagger definition.
```

Comment:
All these PRs had all the check-boxes checked. Maybe instead of doing the blame game though, maybe we should check if the PR check list had the expected outcome and made it easy for reviewers to go over the TODO list.

CC: @firecracker-microvm/compute-capsule what do you think?

Reply:
> CC: @firecracker-microvm/compute-capsule what do you think?

I think that the PR checklist raises the barrier to entry for new contributors and constantly annoys at least one existing contributor (i.e. me).